spec/rubocop/cop/performance/use_zip_to_wrap_array_contents_spec.rb

```diff
@@ -0,0 +1,170 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Performance::UseZipToWrapArrayContents, :config do
+  context 'when using map with array literal' do
+    it 'registers an offense' do
+      expect_offense(<<~RUBY)
+        [1, 2, 3].map { |id| [id] }
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use `.zip` instead of `.map { |id| [id] }`.
```

Comment:
This offense range is a bit too long. It should only start at `.map`.

Reply:
Thanks! What's the mechanism for shortening the offense range?